lib/commands/build-command-helper.ts

```diff
@@ -90,7 +90,19 @@ export class BuildCommandHelper implements IBuildCommandHelper {
 			password = await this.$prompter.getPassword("Apple ID password");
 		}
 
-		return { username, password };
+		return {
+			username,
+			password
+		};
+	}
+
+	public async getExtendedAppleCredentials(args: string[], options: ICloudOptions): Promise<IPublishCredentials> {
+		const credentials = await this.getAppleCredentials(args);
```

Comment:
We should not ask for getAppleCredentials if options.appleSessionBase64

Reply:
🎣